code/modules/clothing/suits/f13suits.dm

```diff
@@ -517,6 +517,7 @@
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/small
 
 /obj/item/clothing/head/hooded/followerlight
+	name = "Follower's Labcoat hood"
```

Comment:
Item names shouldn't be capitalised unless they're proper nouns. "Dr. Bob's Hood" should be capitalised, "follower doctor's hood" should not be. Also, try to avoid using apostrophes/single quotes in item names; `follower doctor hood` is fine.